OpenRA.Mods.RA/Effects/GpsSatellite.cs

```diff
@@ -36,8 +41,12 @@ public void Tick(World world)
 			anim.Tick();
 			pos += new WVec(0, 0, 427);
 
-			if (pos.Z > pos.Y)
+			if (++tick >= revealDelay)
```

Comment:
`++foo` increments the value before returning it, so this will run immediately if `revealDelay` is 1.  Is this what you intended?  I would normally expect this to look like either `if (tick++ >= revealDelay)` or `if (++tick > revealDelay)`.